### ISSUE TITLE
[dotnet-trace] Some rundown events are missing

### ISSUE THREAD

**Issue by kouvel** [dotnet/coreclr#25014]
Examples:
- `Method/DCStopVerbose`
- `Method/ILToNativeMapDCStop`

That may be why when attaching to a process that has completed all of its jitting, the stack view does not show up. I believe `DCStopVerbose` is used by PerfView to associate a sample's code address with a method name and other info about the method, when the method was already jitted before attach, and `ILToNativeMapDCStop` when paired with `DCStopVerbose` is used to associate samples to source code lines.

Some more discussion in https://github.com/dotnet/coreclr/pull/24993

**Comment by tommcdon** [comment]
@davmason can you investigate?

**Comment by davmason** [comment]
I'll take a look in the morning

**Comment by josalem** [comment]
Taking a look at a trace generated using master, I am only seeing `MethodILToNativeMap` events, and no `MethodDC*` events

```
Metrics:
        MethodILToNativeMap:     47
        MethodDCStartV2:         0
        MethodDCStartVerboseV2:  0
        MethodDCStartCompleteV2: 0
        MethodDCStopV2:          0
        MethodDCStopVerboseV2:   0
        MethodDCStopCompleteV2:  0
```

I'm going to take a look at the Rundown and see what might be causing this to happen.

CC @sywhang @jorive

**Comment by sywhang** [comment]
Thanks @josalem! Keep me updated before you leave next week :-)

**Comment by josalem** [comment]
I went back and compared preview 5, 6, and 7 trace outputs.  It would seem we stopped getting these events _between_ previews 5 and 6.

**Comment by jorive** [comment]
@josalem does it work in `preview 7`?

**Comment by josalem** [comment]
There are no `Method*` events showing up in the traces from preview7.

**Comment by sywhang** [comment]
So @josalem and I were able to narrow it down to this commit: https://github.com/dotnet/coreclr/commit/451fd23fa2179fa22c831bcf84edd547bfba5cd9, but the change is quite large and I haven't been able to fully narrow it down what specifically broke it. I'll investigate further after the remainder of training today.

**Comment by janvorli** [comment]
cc: @jkotas

**Comment by jkotas** [comment]
#25199 should fix this

**Comment by jkotas** [comment]
Note fully fixed yet. From https://github.com/dotnet/coreclr/pull/25199#issuecomment-502735310:

After typing that up, I ran my test with an ASP.NET Core WebApi and I am only seeing partial symbols still. Far more than before, but not everything.

I do not see anything obvious that can be causing this.

**Comment by josalem** [comment]
Quick update on this:  Seems like there were a couple issues at work here
1. The one Jan fixed (accidental `#ifdef` placement)
2. Some rundown events were erroneously marked as not enabled (fixed by #25361)
3. Generic R2R code was not emitting the events necessary for symbol resolution (fix coming from @davmason #soon)

With all these things fixed, we get essentially all symbols in my smoke test (an asp.net core webapi):
![image](https://user-images.githubusercontent.com/20667293/60056532-100c1980-9696-11e9-9a6e-959bbe1d1c39.png)

**Comment by davmason** [comment]
Now that #25371 is in, this should be resolved
